ogc/formats/om.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Observations and Measurements
+layout: spec
+author: OGC
+summary: Format definition by the Open Geospatial Consortium that defines 
+---
+
+Goals
+=====
+
+A representation of sensor data (xml) exchanged in a [Sensor Observation Service](../protocols/sos.md)
```

Comment:
internal references are a little awkward, they'll need to end in `.html` rather than `.md` unfortunately to match how the site is set up.